src/test/groovy/graphql/nadel/engine/OverallQueryTransformerTest.groovy

```diff
@@ -127,14 +131,22 @@ class OverallQueryTransformerTest extends Specification {
             fragment frag3 on Foo {
                 qux
             }
+            
+            fragment barFrag on Bar {
+                barId: id
+            }
             ''')
 
         when:
         def delegateQuery = doTransform(schema, query)
 
         then:
         AstPrinter.printAstCompact(delegateQuery) ==
-                "query {foo(id:\"1\") {...frag1}} fragment frag1 on Foo {id ...frag2} fragment frag2 on Foo {bazId ...frag3} fragment frag3 on Foo {qux}"
+                'query {foo(id:"1") {...frag1} bar(id:"1") {...barFrag}} ' +
+                'fragment frag1 on Foo {id ...frag2} ' +
+                'fragment barFrag on Baz {barId:id} ' +
+                'fragment frag2 on Foo {bazId ...frag3} ' +
+                'fragment frag3 on Foo {qux}'
     }
```

Comment:
Nice readability improvement

FYI I learnt you do `\` in groovy comments so that new lines dont count

```
def s = '''some string \
that has no newlines \
but is over N source code lines '''
```